src/Grid/Paginator.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { TablePagination } from 'material-ui/Table';
+import TablePaginationActions from './TablePaginationActions';
+import { withStyles } from 'material-ui/styles';
+
+const styles = theme => ({
+  spacer: {
+    flex: 0
+  },
+  select: {
+    border: '1px solid #c3c3c3',
+    borderRadius: '5px'
+  }
+});
+
+const Paginator = ({ handlePager, rowsPerPage, page, filteredRecordCount, classes, totalRecordCount }) => {
+  const handleChangePage = (event, page) => {
+    handlePager(rowsPerPage, page);
+  };
+    
+  const handleChangeRowsPerPage = event => {
+    handlePager(event.target.value, page);
+  };
+
+  const message = totalRecordCount === filteredRecordCount ?
```

Comment:
this is painful. Add  ({ from, to, count }) as function signature and then the ternary op.